lib/typescript/httpclient/model/Content.ts

```diff
@@ -0,0 +1,13 @@
+import {Template} from './Template';
+import {Message, SenderType} from './Message';
+
+export interface Content {
+  id: string;
+  content: any;
+}
+
+export type RenderedContentUnion = Message | Template;
+
+export function isFromContact(message: RenderedContentUnion) {
+  if (message && 'senderType' in message) return message?.senderType === SenderType.sourceContact;
+}
```

Comment:
typing for render library: _RenderedContentUnion_ represents the type for the content that the library is rendering